[DOCS] Gives more details to the load data step of the semantic search tutorials (#113088)
elastic/elasticsearch: docs/reference/search/search-your-data/semantic-search-elser.asciidoc

> It is only used in this tutorial as a sample dataset that is easily accessible for demonstration purposes.

Reviewer: ```suggestion
We use this sample dataset in the tutorial because is easily accessible for demonstration purposes.
```

🚗

You can use a different data set to test the workflow and become familiar with it.

> Download the file and upload it to your cluster using the {kibana-ref}/connect-to-elasticsearch.html#upload-data-kibana[Data Visualizer] in the {ml-app} UI.

Reviewer: ```suggestion
Download the file and upload it to your cluster using the {kibana-ref}/connect-to-elasticsearch.html#upload-data-kibana[File Uploader] in the UI.
```

Search team have been using **File Uploader** term, so probably makes sense to align with that usage. See also [ES3 docs](https://www.elastic.co/docs/current/serverless/elasticsearch/ingest-data-file-upload).

Also minor suggestion to say "UI" instead of "ML UI"

After your data is analyzed, click **Override settings**.
Under **Edit field names**, assign `id` to the first column and `content` to the second.
Click **Apply**, then **Import**.
Name the index `test-data`, and click **Import**.

> Once the upload is complete, you will see an index named `test-data` with 182,469 documents.

Reviewer: ```suggestion
After the upload is complete, you will see an index named `test-data` with 182,469 documents.
```

Shaina and Marci have been beating the use of "once" out of me 🫨 😄

You can also open the Trained Models UI, select the Pipelines tab under ELSER to follow the progress.

> Following this tutorial, you can also cancel the reindexing process if you don't want to wait until the reindexing process is fully complete which might take hours for large data sets.
> You can test the feature even if you reindex only a subset of the data set - a few thousand data points for example - and generate embeddings for the subset.

Reviewer: ```suggestion
Reindexing large datasets can take a long time. You can test this workflow using
only a subset of the dataset. Do this by cancelling the reindexing process, and
only generating embeddings for the subset that was reindexed.
```
